system-int-test/test/cmr/system_int_test/search/tagging/tag_association_collection_revisions_test.clj

```diff
@@ -56,107 +56,108 @@
 
     (testing "successful case, the tag association keys can have either _ or -"
       (let [response (tags/associate-by-concept-ids
-                       token tag-key [{:concept_id (:concept-id coll1-1)
```

Comment:
James is going to be very unhappy about all of these lines modified to remove a space.